### DISCUSSION TITLE
Side modal Evergreen to Paste discrepancy

### DISCUSSION THREAD

**Discussion by charleneyjh1** [twilio-labs/paste#3288] [Q&A] [answered]
Hello team, there are some discrepancy of the the side modal component for Evergreen-> Paste migration and I just wanna double check if this is the expected behavior.

1. Evergreen has an overlay for the side modal and Paste doesn't
2. For Evergreen, user can close the side modal when clicking outside of the side modal, where for Paste user can only close the side modal when clicking "x".

[Evergreen documentation](https://evergreen.segment.com/components/side-sheet) for reference

Are those the expected behavior of side modal for Paste? I'm aware the side modal component is still going through peer review, so just wanna check here.

**Comment by serifluous** [comment, marked as answer]
Hey @charleneyjh1, yep, both are intentional!

1. The purpose of using a modal that appears on the side (vs. using one in the center) is so that users can still view what's underneath, so we did away with the overlay.
2. Since the content underneath is visible, we decided not to block interaction with that content, so the only way to close the side modal is through a button in the side modal.

And yep, the component still needs to go through peer review, so we'd love to hear any customer issues that come up because of this behavior.

**Reply by charleneyjh1** [reply]
Thank you so much for confirming! @serifluous
